python/pyarrow/table.pxi

```diff
@@ -99,6 +99,14 @@ cdef class ChunkedArray:
             for item in chunk:
                 yield item
 
+    def __richcmp__(ChunkedArray self, ChunkedArray other, int op):
+        if op == cp.Py_EQ:
+            return self.equals(other)
+        elif op == cp.Py_NE:
+            return not self.equals(other)
+        else:
+            raise TypeError('Invalid comparison')
```

Comment:
Could You please implement `__eq__` instead of `__richcmp__`? 
Similarly like: https://github.com/apache/arrow/blob/master/python/pyarrow/types.pxi#L134-L138

`Column.__richcmp__` should be rewritten to `__eq__` as well.

Reply:
OK no problem, thank you!